archive.php

```diff
@@ -13,8 +13,9 @@
 	<div id="primary" class="content-area">
 		<main id="main" class="site-main">
 
-		<?php if ( have_posts() ) : ?>
-
+		<?php if ( have_posts() ) :
+            echo get_bezirksseiten_header(get_the_category());
+            ?>
```

Comment:
Ich hab's jetzt nicht lokal bei mir laufen, also verzeih mir bitte, falls es eine blöde Frage ist, aber:
Die Idee ist den Header nicht mehr in der Page/im Post zu haben, sondern direkt im Theme, right? In dem Header sind aber derzeit mehr als nur das Hintergrundbild, sondern auch diese Box (mit unterschiedlichem Text je Bezirk) inkl. Newsletter Sign Up. Wie kommt dieser Content dann wieder in den Header rein?

Reply:
Keine blöde Frage :) es geht dabei nicht um die Bezirks-Seiten selbst sondern um die Seiten, auf denen ein einzelner Post oder aber das Post-Archiv angezeigt wird (wenn man auf eine Kategorie klickt). Da gab es bis jetzt noch gar keinen Header. Ich hab das mal nur in der minimal-Version gemacht, wär auch kein großes Ding da auch die Formulare und den Text dazu zu tun.